local/bin/py/build/configurations/pull_config_preview.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 - config:
-    cache_enabled: true
+    cache_enabled: false
```

Comment:
```suggestion
    cache_enabled: true
```

Reply:
@alai97 setting this to true on preview reverts to a cached version that is not desired. i'll try make another PR that sets the cache to true after it is merged to master if that is ok